ansible-role-requirements.yml

```diff
@@ -14,3 +14,7 @@
   scm: git
   src: https://github.com/rsoprivatecloud/openstack-ops
   version: 1383d0db302c6fc400a50fae4f85e77bf1633581
```

Comment:
Can we add a note about this needing to come out, as exists here:

https://github.com/rcbops/rpc-openstack/blob/newton-14.1/ansible-role-requirements.yml#L21-L23